src/main/java/com/demoipm/dto/general/ResponseDto.java

```diff
@@ -0,0 +1,30 @@
+package com.demoipm.dto.general;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.io.Serializable;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ResponseDto implements Serializable {
```

Comment:
Why do you use the 
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonIgnoreProperties(ignoreUnknown = true)
here? Can do you explain?

Reply:
This one is just in case the object transfer from json, it will take only the field that are define as property of object and ignore all unknown field.
If dont use @JsonIgnoreProperties(ignoreUnknown = true), when json have field which wasn't defined in object, we will receive error when transform json to the object

Reply:
I'm a little confused in here because we doesn't using API in this project.